macros/src/lib.rs

```diff
@@ -322,20 +332,18 @@ pub fn option_setters(input: proc_macro::TokenStream) -> proc_macro::TokenStream
         setters,
     } = parse_macro_input!(input as OptionSettersList);
 
-    let extras = opt_field_name.map(|name| {
-        quote! {
-            #[allow(unused)]
-            fn options(&mut self) -> &mut #opt_field_type {
-                self.#name.get_or_insert_with(<#opt_field_type>::default)
-            }
+    let extras = quote! {
```

Comment:
There was leftover unused functionality to omit the `options` and `with_options` from generated code; stripping that out let me simplify assumptions for the newer iteration.